_posts/2022-11-08-pandas-dataframe-output-for-sklearn-transformer.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Pandas DataFrame output for Sklearn Transformers"
+date: November 8, 2022
+categories:
+  - Technical
+tags:
+  - performance
+featured-image: pandas_output_sklearn_transformers.PNG
+
+postauthors:
+  - name: Sangam Swadi K
+    website: https://www.linkedin.com/in/sangam-swadi-k/
+    image: sangam_swadik.jpg 
+---
+
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+## Video
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5bCg8VfX2x8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Upcoming feature in release 1.2
+Starting with the next release of [scikit-learn](https://github.com/scikit-learn/scikit-learn) (v1.2), pandas dataframe output will be available for all sklearn transformers! This will make running pipelines on dataframes much easier and provide better ways to track feature names. Previously, mapping a transformed output back into columns would be cumbersome as it might not be a one-to-one mapping in cases of complex preprocessing (e.g., polynomial features ).
```

Comment:
```suggestion
Starting with the next release of [scikit-learn](https://github.com/scikit-learn/scikit-learn) (v1.2), pandas dataframe output will be available for all sklearn transformers! This will make running pipelines on dataframes much easier and provide better ways to track feature names. Previously, mapping a transformed output back into columns would be cumbersome as it might not be a one-to-one mapping in cases of complex preprocessing (e.g., polynomial features).
```